gm4_template_pack/data/gm4/advancements/MODULE_ID.json

```diff
@@ -1,19 +1,29 @@
 {
-  "display":{
-    "icon":{
-      "item":"stone",
-      "nbt":"{display:{Name:'\"MODULE_ID logo\"'}}"
+  "display": {
+    "icon": {
+      "item": "stone",
+      "nbt": "{CustomModelData:1}"
     },
-    "title":"Plain Ol' Advancement Title",
-    "description":{
-      "text":"Noone's bothered to give me a description yet... They won't... They don't care about me....",
-      "color":"gray"
+    "title": {
+      "translate": "%1$s",
+      "with": [
+        "Plain Ol' Advancement Title",
+        {"translate": "advancement.gm4.template_pack.title"}
```

Comment:
In theory, shouldn't this and its description be `advancement.gm4.MODULE_ID` not `advancement.gm4.template_pack`? It just helps with a mass replace when using the template

Reply:
I just realised, in the resource pack you have the language string for the template pack, but no-one sees these dummy strings anyways?

Reply:
All folders need to be lowercase so that wouldn't work